policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal/amount.yaml

```diff
@@ -12,27 +12,17 @@ metadata:
     href: https://houpl.org/wp-content/uploads/2023/01/2022-IT-511_Individual_Income_Tax_-Booklet-compressed.pdf#page=12
   - title: 2021 Form 500 Instructions
     href: https://houpl.org/wp-content/uploads/2022/01/2021_IT-511_Individual_Income_Tax_Booklet-12.29.21-Web-Version.pdf#page=13
+  # The personal exemptions were repealed in 2024
```

Comment:
This statute defining the amount remains active though right? It's a bit weird but I think we should leave it in if so.

Reply:
Justia might be behind here:
I dint don't see the section § 48-7-26 in Casetext:
https://casetext.com/statute/code-of-georgia/title-48-revenue-and-taxation/chapter-7-income-taxes/article-2-imposition-rate-and-computation-exemptions

Reply:
Could you replace the Justia ref with Casetext then? They do have 48-7-26 but not the personal piece.

Reply:
done